samples/health-check/liveness-http-same-port.yaml

```diff
@@ -15,6 +15,9 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: liveness-http
+  labels:
+    app: liveness-http
+    version: v1
```

Comment:
For consistency between all the pages, do we want the metadata name to be [app label]-[app version]?

In this case, we remove the app version or make the metadata name `liveness-http-v1`

The same comment would apply in other files.

Reply:
> For consistency between all the pages, do we want the metadata name to be [app label]-[app-version]?

I think that will be good to have. Not sure if that also requires doc updates wherever we use these labels?

Reply:
Hence, maybe removing the version if it's not specified. I think this is making the added label(s) match the current metadata name.

Reply:
> I think this is making the added label(s) match the current metadata name.

Please see https://github.com/istio/istio/pull/27242#issuecomment-691468933

Reply:
Removed based on https://github.com/istio/istio/pull/27242#issuecomment-692549416